### ISSUE TITLE
Add termenv's SetWindowTitle to BubbleTea?

### ISSUE THREAD

**Issue by erwin** [charmbracelet/bubbletea#610]
In BubbleTea there are wrappers for many of escape sequences that [termenv](https://github.com/muesli/termenv) provides (show/hide cursor, enter/exit alt screen, clearScreen).

I think it would be also be nice to include `termenv`s `SetWindowTitle` in BubbleTea.

It's easy enough to just output the escape codes to switch the title in your view, but I think adding this one would be in the spirit of the rest of BubbleTea.

```
"\x1b]2;" + YourTitle + "\a" 
```

Ideally I would like to trigger title changes periodically based on important state changes identified during `Update()` rather than splicing it into the View.

**Comment by meowgorithm** [comment]
Yep, this is a feature we’d like to get in. Because of the i/o involved it should be implemented as a `Cmd`.

**Comment by uaealbloshi2** [comment]
[]()
